NewForce-Cohort-9/tabloid-full-stack-full-stack-force
# Fw ticket36 upload user image

Description
Added file upload feature for user profile image. [Trello Ticket #36](https://trello.com/c/MCsDNcGa/36-36-upload-user-profile-image)


Type of Change
-Added new, non-breaking feature.


Testing Instructions
1. git add and git commit whatever you're working on on your branch
2. git checkout main
3. git fetch --a
4. git checkout fw-ticket36-uploadUserImage
5. If you don't see the most recent version of my files, git pull origin fw-ticket36-uploadUserImage
6. If you haven't created the Tabloid database, please do so.
7. Use this to fill out tables and columns: [SQL data](https://github.com/NewForce-Cohort-9/tabloid-full-stack-full-stack-force/blob/main/SQL/01_Tabloid_Create_DB.sql)
8. Use this to seed the database: [SQL Seed Data](https://github.com/NewForce-Cohort-9/tabloid-full-stack-full-stack-force/blob/main/SQL/02_Tabloid_Seed_Data.sql)
9. Inside of Visual Studio (purple), click the execute with debugger solid green button to launch the app
10. In your terminal, navigate to where your package.json file is in your client folder. Run "npm start" in your terminal.
11. Login as an admin. foo@bar.com for the email parameter. Leave the password text box blank. Click "Login".
12. Click on User Profiles at the top of the navbar.
13. Once on the User Profiles page, it should list User Profiles. For the first few profiles, I have added images. Try adding your own for [rsandwith0](http://localhost:3000/profile/2).
14. Click on the user profile Display Name and you will see the user's profile details as well as “Choose File” and “Upload” buttons on the left of the page above the user’s Display Name. Click on “Choose File” button and choose an image from your computer, then click on “Upload”.
15. The page should automatically refresh and display the picture you uploaded.

Checklist:
[x] My code follows the style guidelines of this project
[x] I have performed a self-review of my code
[x] I have commented my code, particularly in hard-to-understand areas
[x] My changes generate no new warnings or errors
[x] I have added test instructions that prove my fix is effective or that my feature works
